internal/utils/string.go

```diff
@@ -0,0 +1,9 @@
+package utils
+
+// CoalesceString returns empty string on null, otherwise returns the value
+func CoalesceString(value *string) string {
```

Comment:
I added this little func to help me avoid splattering the code with if/else. Let me know what you think

Reply:
It's fine